### PR TITLE
SAK-47640 Rubrics Fix: Total Grade not updating correctly

### PR DESCRIPTION
**Pull request by V100X**
Fix for [#10776](https://github.com/sakaiproject/sakai/pull/10776)

Total Grade for weighted Rubrics:
-  wasn't being calculated correctly when using Criterion Groups.
-  wasn't updating when Ratings where added or removed.

### CODE REVIEW

File: webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-criteria.js [hunk 1]
```diff
@@ -444,6 +443,7 @@ export class SakaiRubricCriteria extends RubricsElement {
     const criterion = this.criteria.find(c => c.id == criterionId);
 
     totalPoints += calculateCriteriaPoints(criterion, minOrMax);
+    this.dispatchEvent(new CustomEvent('refresh-total-weight', { detail: { criteria: this.criteria } }));
```
Comment (mpellicer) [inline webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-criteria.js:446]: Could you confirm this works when deleting a rating, adding a rating, saving a new score for the rating? 

Anyway, I'm going to test these changes locally.

Comment (V100X) [reply webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-criteria.js:446]: It works for the actions you mentioned ✅

**Comment by mpellicer** [comment]
I see a JS error, could you confirm?
Sakai  abcde  Tests   Quizzes - Google Chrome

**Comment by V100X** [comment]
I see it too, trying to understand the problem.

File: webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-criteria.js (hunk 1 above)
Comment (mpellicer) [reply webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-criteria.js:446]: @V100X Unfortunatelly I still see the issue when deleting a rating, it states 0 points for the criterion. Added a GIF with the issue, when deleting the rating the maximum score of the criterion goes to 0.

SAK-47640

Comment (mpellicer) [reply webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-criteria.js:446]: Same happens with the min value when removing the lowest rating.
SAK-47640-2

**Review by mpellicer — CHANGES_REQUESTED** [review]

File: webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-criteria.js (hunk 1 above)
Comment (V100X) [reply webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-criteria.js:474]: Works well on my local, anyway, hopefully it's solved now

**Review by mpellicer — APPROVED** [review]
All issues fixed and testing went well locally, well done @V100X !

**Pull request merged**